src/Nix/Diff/Transformations.hs

```diff
@@ -0,0 +1,87 @@
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE MultiWayIf #-}
+module Nix.Diff.Transformations where
+
+import Nix.Diff.Types
+
+{-| In large diffs there may be a lot of derivations
```

Comment:
Yes, we can use example from readme for this purpose
```
29,31c29
<       • The input derivation named `dbus-1` differs
<         • These two derivations have already been compared
<       • Skipping environment comparison
---
>       • Skipping because only derivations that have already been compared and shown in the diff are below
35,37c33
<       • The input derivation named `system-path` differs
<         • These two derivations have already been compared
<       • Skipping environment comparison
---
>       • Skipping because only derivations that have already been compared and shown in the diff are below
41,43c37
<       • The input derivation named `system-path` differs
<         • These two derivations have already been compared
<       • Skipping environment comparison
---
>       • Skipping because only derivations that have already been compared and shown in the diff are below
80,86c74
<     • The input derivation named `unit-dbus.service` differs
<       - /nix/store/1h7np4jmaf2vc6sczvj7sg1hhinh2lzq-unit-dbus.service.drv:{out}
<       + /nix/store/llg852i3p19q17hw98gq11r61wbvjnrj-unit-dbus.service.drv:{out}
<       • The input derivation named `dbus-1` differs
<         • These two derivations have already been compared
<       • Skipping environment comparison
<     • Skipping environment comparison
---
>     • Skipping because only derivations that have already been compared and shown in the diff are below
```